### ISSUE TITLE
How to fine-tune the pretrain-model

### ISSUE THREAD

**Issue by Hyfred** [audeering/w2v2-how-to#18]
I was wondering how to fine-tune the released model in another dataset.

**Comment by hagenw** [comment]
For fine-tuning you can download the torch version of the model from https://huggingface.co/audeering/wav2vec2-large-robust-12-ft-emotion-msp-dim.

We mention in the README that the torch model is published there:

![image](https://user-images.githubusercontent.com/173624/189088944-4d52d874-95fe-4650-99e3-8bcf84b10511.png)

but maybe we should highlight this more?

**Comment by Vedaad-Shakib** [comment]
Some details like the Adam optimizer hyperparameters are not given in the paper. Should we assume that you use the default Wav2Vec2 hyperparameters if not specified in the paper?

**Comment by frankenjoe** [comment]
Yes, if not mentioned otherwise we kept to the default parameters of [TrainingArguments](https://huggingface.co/docs/transformers/v4.36.1/en/main_classes/trainer#transformers.TrainingArguments).
